lib/requestability_resolver.js

```diff
@@ -1,21 +1,43 @@
 const DeliveryLocationsResolver = require('./delivery-locations-resolver')
+const { isItemNyplOwned } = require('./ownership_determination')
+const { isInRecap } = require('./util')
+const logger = require('./logger')
 class RequestabilityResolver {
   static fixItemRequestability (elasticSearchResponse) {
     elasticSearchResponse.hits.hits
       .map((hit) => {
         hit._source.items = hit._source.items.map((item) => {
           if (item.electronicLocator) return item
           let deliveryInfo
-          if (item.recapCustomerCode && item.recapCustomerCode[0]) {
+          const itemIsInRecap = isInRecap(item)
+          let physRequestableCriteria
+          if (itemIsInRecap && !item.recapCustomerCode) {
```

Comment:
As discussed, seems like a `hasRecapCustomerCode` var may be warranted to simplify these checks (i.e. `const hasRecapCustomerCode = item.recapCustomerCode && item.recapCustomerCode[0]`) so that you can make the if/else flow more clearly about three disjoint scenarios:
```
if (itemIsInRecap && !hasRecapCustomerCode) {
  ...
} else if (itemIsInRecap && hasRecapCustomerCode) {
  ...
} else { // item is not in ReCAP
  
}
```